docker/puppet-agent-alpine/Dockerfile

```diff
@@ -8,32 +8,32 @@ RUN sed -i -e 's/sys\/poll/poll/' /usr/include/boost/asio/detail/socket_types.hp
 
 RUN git clone -b 1.5.x https://github.com/puppetlabs/leatherman && mkdir -p /workspace/leatherman/build
 WORKDIR /workspace/leatherman/build
-RUN cmake  -DBOOST_STATIC=OFF -DCMAKE_VERBOSE_MAKEFILE=ON ..; make ; make install
+RUN cmake -DCMAKE_PREFIX_PATH=/opt/puppetlabs/puppet -DCMAKE_INSTALL_PREFIX=/opt/puppetlabs/puppet -DCMAKE_INSTALL_RPATH=/opt/puppetlabs/puppet/lib -DBOOST_STATIC=OFF -DCMAKE_VERBOSE_MAKEFILE=ON ..; make ; make install
```

Comment:
Can we setup global defaults for these... or better to be explicit on each call?

Reply:
I think setting up defaults would be reasonable, will take a look today.